wagtailio/project_styleguide/templates/patterns/components/streamfields/quotes.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailcore_tags %}
+
+<div class="quotes">
```

Comment:
This will need the `grid` class adding to it to align it when it sits on the page - see https://github.com/wagtail/wagtail.org/blob/next/wagtailio/static/sass/components/_grid.scss

Reply:
This feels like a good one for FE developer documentation (hint, hint :D)

Reply:
I'll add a ticket : )

Reply:
thanks both!